rome-core/src/main/java/com/rometools/rome/feed/rss/Enclosure.java

```diff
@@ -30,7 +30,7 @@
 public class Enclosure implements Cloneable, Serializable {
     private static final long serialVersionUID = 1L;
     private String url;
-    private long length = -1L;
+    private Long length = Long.valueOf(-1L);
```

Comment:
As mentioned at https://github.com/rometools/rome/pull/582#discussion_r1002159048, that was not my recommendation :)